[BUGFIX] Display author URI as an external link

beatrycze-volk opened the pull request:
Related to #1567

beatrycze-volk committed:
Display author URI as an external link

beatrycze-volk committed:
Add missing `dd` tag for external links

sebastian-meyer committed:
Merge branch '5.0.x' into display-author-uri

sebastian-meyer commented on Classes/Format/Mods.php:
```diff
@@ -173,11 +173,16 @@ private function getAuthorFromXml(array $authors, int $i): void
         if (isset($authors[$i]['valueURI'])) {
-            $this->metadata['author'][$i] .= pack('C', 31) . (string) $authors[$i]['valueURI'];
+            $displayName = $this->metadata['author'][$i];
+            $this->metadata['author'][$i] = [
+                'name' => $displayName,
+                'url' => (string) $authors[$i]['valueURI']
+            ];
```
How does this affect the indexing of author names? You adjusted the templates for display, but I don't see any changes to the indexing routines. Are those not affected?

beatrycze-volk replied:
Thanks for comment. I didn't consider this case. I will look into it.

beatrycze-volk committed:
Changes to displaying external links in metadata

- use Kitodo wrapper
- remove `externalUrl` key from array
- allow more then one author / holder

beatrycze-volk committed:
Get only names for storing authors in database

sebastian-meyer committed:
Merge branch '5.0.x' into display-author-uri

sebastian-meyer reviewed (approved):
Thanks you!
